ansible/roles/kubernetes/tasks/main.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: debian family specific tasks
+  include_tasks: debian.yml
+  when: ansible_os_family | lower == 'debian'
+
+- name: archlinux family specific tasks
+  include_tasks: archlinux.yml
+  when: ansible_os_family | lower == 'archlinux'
+
 - name: Reset Kubernetes Instance (1/2)
```

Comment:
Is there some condition missing here? Looks like the reset will now be done each time the role is executed. Does not seem like a good plan to me, will destroy your nodes.

Reply:
Good point, currently the logic doesn't support adding nodes to the pool once an entire cluster setup has been completed, at the moment it's assuming a complete cluster rebuild each time this is executed, although Ideally, we don't want to have to do that if possible.